src/app/clusters/bindings/BindingManager.cpp

```diff
@@ -161,6 +161,15 @@ void BindingManager::FabricRemoved(CompressedFabricId compressedFabricId, Fabric
     mAppServer->GetCASESessionManager()->ReleaseSessionForFabric(compressedFabricId);
 }
 
+CHIP_ERROR BindingManager::NotifyBindingAdded(const EmberBindingTableEntry & binding)
+{
+    if (mBindingAddedHandler)
+    {
+        mBindingAddedHandler(binding);
```

Comment:
should the handler have the option to return a CHIP_ERROR? or do we need that notify always returns CHIP_NO_ERROR?

Reply:
there is no need for the callback to return a value since its a notification